### PR TITLE
Store the CLA text with the agreement model

### PR DESCRIPTION
**Pull request by teone**
…greementForm

### CODE REVIEW

**Comment by ccascone** [comment]
@teone pro tip - mentioning the issue number in the PR title doesn't do much, instead:
https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

File: client/src/js/agreement/AgreementForm.jsx [hunk 1]
```diff
@@ -61,13 +79,13 @@ function AgreementForm (props) {
...
         props.agreementType === AgreementType.INDIVIDUAL ? AgreementType.INDIVIDUAL : AgreementType.CORPORATE,
-        'TODO, add agreement body',
+        ClaText,
         signer
       )
     } else if (props.agreementType === AgreementType.CORPORATE) {
       agreement = new Agreement(
         props.agreementType === AgreementType.INDIVIDUAL ? AgreementType.INDIVIDUAL : AgreementType.CORPORATE,
-        'TODO, add agreement body',
+        ClaText,
```
Comment (ccascone) [inline client/src/js/agreement/AgreementForm.jsx]: Is the corporate text expected to be the same as the individual one? If not, we should open a GitHub issue to add both.

File: client/src/js/agreement/AgreementDisplay.jsx [hunk 2]
```diff
@@ -1,104 +1,45 @@
...
-        details of its source and of any license or other restriction (including, but not limited to, related patents,
-        trademarks, and license agreements) of which you are personally aware, and conspicuously marking the work as
-        "Submitted on behalf of a third-party: [named here]".
-      </p>
-      <p>8. <b>Required Notifications. </b>You agree to notify ONF of any facts or circumstances of which you become
-        aware that would make these representations inaccurate in any respect.
-      </p>
+      {
+        props.text ?
+          <div className={classes.claContainer} dangerouslySetInnerHTML={{ __html: props.text }}/>
```
Comment (ccascone) [inline client/src/js/agreement/AgreementDisplay.jsx]: If storing HTML is so dangerous, would it make sense to store markdown and use a renderer such as this one: https://github.com/rexxars/react-markdown

File: client/src/js/agreement/AgreementForm.jsx (hunk 1 above)
Comment (teone) [reply client/src/js/agreement/AgreementForm.jsx]: I thought the multiple CLA was not to be addressed for this release and tracked here: https://github.com/OpenNetworkingFoundation/cla-manager/issues/41

But we can probably have two different ones for Individual and Corporate

File: client/src/js/agreement/AgreementDisplay.jsx (hunk 2 above)
Comment (teone) [reply client/src/js/agreement/AgreementDisplay.jsx]: Nice find.. I'll add `rect-markdown`

File: client/src/js/agreement/AgreementForm.jsx [hunk 3]
```diff
@@ -79,13 +86,13 @@ function AgreementForm (props) {
     if (props.agreementType === AgreementType.INDIVIDUAL) {
       agreement = new Agreement(
         props.agreementType === AgreementType.INDIVIDUAL ? AgreementType.INDIVIDUAL : AgreementType.CORPORATE,
```
Comment (ccascone) [inline client/src/js/agreement/AgreementForm.jsx]:
```suggestion
        AgreementType.INDIVIDUAL,
```

File: client/src/js/agreement/AgreementForm.jsx [hunk 4]
```diff
@@ -79,13 +86,13 @@ function AgreementForm (props) {
...
-        ClaText,
+        ClaTextIndividual,
         signer
       )
     } else if (props.agreementType === AgreementType.CORPORATE) {
       agreement = new Agreement(
         props.agreementType === AgreementType.INDIVIDUAL ? AgreementType.INDIVIDUAL : AgreementType.CORPORATE,
```
Comment (ccascone) [inline client/src/js/agreement/AgreementForm.jsx]:
```suggestion
        AgreementType.CORPORATE,
```

**Pull request merged**